src/main/java/com/denizenscript/depenizen/bukkit/commands/mythicmobs/MythicSignalCommand.java

```diff
@@ -42,45 +41,11 @@ public MythicSignalCommand() {
     //
     // -->
 
-    @Override
-    public void parseArgs(ScriptEntry scriptEntry) throws InvalidArgumentsException {
-        for (Argument arg : scriptEntry) {
-            if (!scriptEntry.hasObject("targets")
-                    && arg.matchesArgumentList(MythicMobsMobTag.class)) {
-                scriptEntry.addObject("targets", arg.asType(ListTag.class).filter(MythicMobsMobTag.class, scriptEntry));
-            }
-            else if (!scriptEntry.hasObject("source")
-                    && arg.matchesPrefix("source")
-                    && arg.matchesArgumentType(EntityTag.class)) {
-                scriptEntry.addObject("source", arg.asType(EntityTag.class));
-            }
-            else if (!scriptEntry.hasObject("signal")) {
-                scriptEntry.addObject("signal", arg.asElement());
-            }
-            else {
-                arg.reportUnhandled();
-            }
-        }
-        if (!scriptEntry.hasObject("targets")) {
-            throw new InvalidArgumentsException("Must specify MythicMobs to send the signal to.");
-        }
-        if (!scriptEntry.hasObject("signal")) {
-            throw new InvalidArgumentsException("Must specify a signal to send.");
-        }
-        if (!scriptEntry.hasObject("source")) {
-            throw new InvalidArgumentsException("Must specify a source entity.");
-        }
-    }
-
-    @Override
-    public void execute(ScriptEntry scriptEntry) {
-        EntityTag source = scriptEntry.getObjectTag("source");
-        List<MythicMobsMobTag> targets = (List<MythicMobsMobTag>) scriptEntry.getObject("targets");
-        ElementTag signal = scriptEntry.getElement("signal");
-        if (scriptEntry.dbCallShouldDebug()) {
-            Debug.report(scriptEntry, getName(), db("mythicmobs", targets), signal, source);
-        }
-        for (MythicMobsMobTag mob : targets) {
+    public static void autoExecute(ScriptEntry scriptEntry,
+                                   @ArgLinear @ArgName("targets") ListTag targets,
```

Comment:
Should use `@ArgSubType(MythicMobsMobTag.class)` and take a `List<MythicMobsMobTag>`